src/vsr/replica.zig

```diff
@@ -8466,7 +8587,11 @@ pub fn ReplicaType(
                 @panic("checkpoint diverged");
             }
 
-            if (candidate.view > self.view_durable()) {
+            // TODO: This block introduced by the PR #1598 causes a liveness issue
+            // when a replica cannot start the sync.
+            // SEED 10266230273529621812, 8842160722467235252
+            const fix_me = true;
+            if (fix_me and candidate.view > self.view_durable()) {
```

Comment:
I still need to investigate this, unsure if it is caused by the `pulse` operations or if they just happened to expose the existing issue.

Reply:
See https://github.com/tigerbeetle/tigerbeetle/pull/1657#discussion_r1523023985